enterprise/backend/src/metabase_enterprise/workspaces/isolation.clj

```diff
@@ -21,15 +22,15 @@
 (defn- isolation-schema-name
   "Generate schema/database name for workspace isolation following mb__isolation_<slug>_<workspace-id> pattern."
   [workspace-id]
-  (let [instance-slug (instance-uuid-slug (str (system/site-uuid)))
+  (let [instance-slug      (instance-uuid-slug (str (system/site-uuid)))
         clean-workspace-id (str/replace (str workspace-id) #"[^a-zA-Z0-9]" "_")]
     (format "mb__isolation_%s_%s" instance-slug clean-workspace-id)))
 
-(defn- isolated-transform-table-name
+(defn- isolated-table-name
```

Comment:
Having this clone a table rather than a transform saves us from making the function polymorphic for each transform type, and may give us useful re-usability if we need to copy input data too (e.g. for Clickhouse)